src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
-'use strict';
+"use strict";
 
 // write your code here
+const populationArray = [...document.querySelectorAll(".population")];
+
+const totalPopulation = populationArray.reduce(
+  (acc, p) =>
+    isNaN(parseInt(p.textContent.replace(/,/g, "")))
```

Comment:
why do we need additional checks for `isNaN` and parseInt?
just use 
```
acc + +value.textContent.split(',').join('')
```

Reply:
Thanks, done!